pegasus/sites.v3/hourofcode.com/i18n/public/sv/promote/country-resources.md

```diff
@@ -140,6 +140,6 @@ p>[**Pedro Pablo Kuczynski. Presidente del Perú 2016-2021**](https://www.youtub
 
 <% else %>
 
-# Se fler övningar och material för Sverige på <edvira.com/hoc>!
+# Se fler övningar och material för Sverige på [edvira.com/hoc](edvira.com/hoc)!
```

Comment:
I'm changing the translation of this in crowdin to remove the link so you don't have to keep fixing this.

Reply:
I think we partner with these folks; it'd be worth confirming with @sukykang that we don't care about that link if you haven't already

Reply:
This seems to be a hidden string on Crowdin. Could you tell me where on hoc.com this shows up? FYI @jakebrbell

Reply:
Sorry, to clarify the translated text still has the link as plaintext. It's no longer hyperlinked. I don't think we should let them put in their own HTML like this, since it obviously breaks our pipeline.

https://hourofcode.com/se/promote/country-resources